lib/rotoscope/call_logger.rb

```diff
@@ -88,34 +102,59 @@ def state
     private
 
     def log_call(call)
-      caller_path = call.caller_path || ''
-      return if blacklist.match?(caller_path)
+      return if blacklist.match?(filepath(call))
       return if self == call.receiver
 
-      caller_class_name = call.caller_class_name || '<UNKNOWN>'
+      buffer = @output_buffer
+      buffer.clear
+
+      if @header.size > 1
+        @header[0...-1].each do |head|
```

Comment:
We don't want to create this `@header[0...-1]` on each log call.  However, that could be moved to the initializer.